src/app/dataset-block/metadata-block/components/event-details/components/common/time-property/time-property.component.ts

```diff
@@ -0,0 +1,12 @@
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { BasePropertyComponent } from "../base-property/base-property.component";
+
+@Component({
+    selector: "app-time-property",
+    templateUrl: "./time-property.component.html",
+    styleUrls: ["./time-property.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TimePropertyComponent extends BasePropertyComponent {
```

Comment:
Can we somehow avoid this wrapper - maybe by unifying the interface of wrapped component?